test/io/testcjsonreader.py

```diff
@@ -24,6 +24,10 @@
 class CJSONReaderTest(unittest.TestCase):
     """Unit tests for the CJSON reader."""
 
+    def test_subclass(self):
+        """Is the reader a subclass of the abstract file reader?"""
+        self.assertTrue(issubclass(cclib.io.cjsonreader.CJSON, cclib.io.filereader.Reader))
```

Comment:
What's your take on using `assertIsInstance` instead? https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance

Reply:
I don't think that's applicable here, since CJSON is a class, not an instance of it.

However, I'm not sure we really need to make this assertion at all, since it's mostly testing Python's implementation and doesn't say anything about the behavior of cclib per se. Instead, is there an assertion we can make about CJSON or an instance of it that relies on it being an implementation of the abstract Reader?